src/Jobs/RemoveStorageSymlinks.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Artisan;
+use Stancl\Tenancy\Contracts\Tenant;
+use Stancl\Tenancy\Events\RemovingStorageSymlink;
+use Stancl\Tenancy\Events\StorageSymlinkRemoved;
+
+class RemoveStorageSymlinks implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    public Tenant $tenant;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Tenant $tenant)
+    {
+        $this->tenant = $tenant;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        event(new RemovingStorageSymlink($this->tenant));
+
+        Artisan::call('tenants:link', [
+            '--remove' => true,
+            '--tenants' => [$this->tenant->getTenantKey()],
+        ]);
+
+        event(new StorageSymlinkRemoved($this->tenant));
```

Comment:
Similar to the review in `CreateStorageSymlinks`. Also firing events *only* in the job and not the command seems weird. The events should be fired as part of the logic, it should be a single atomic thing.